yuptoo/modifiers/transform_cloud_provider.py

```diff
@@ -0,0 +1,20 @@
+from yuptoo.processor.utils import Modifier
+import logging
+
+LOG = logging.getLogger(__name__)
+GOOGLE_PROVIDER_NAME = 'gcp'
+
+
+class TransformCloudProvider(Modifier):
+    def run(self, host: dict, transformed_obj: dict, **kwargs):
+        """Transform 'system_profile.cloud_provider.
+        When system_profile.cloud_provider is 'google',
+        transform it to 'gcp' for consistency.
+        """
+        system_profile = host.get('system_profile', {})
+        cloud_provider = system_profile.get('cloud_provider')
+        if cloud_provider is not None and 'google' in cloud_provider.lower():
```

Comment:
Are we just processing archives from `GCP`?
I think we need to include others as well.

Reply:
As per offline [conversation](https://redhat-internal.slack.com/archives/C018YNTB203/p1693997757723339), the issue is only systems from GCP.